### PR TITLE
feat: introduce network versions 19 and 20

### PR DESCRIPTION
**Pull request by arajasek**
As in the title.

### CODE REVIEW

File: shared/src/version/mod.rs [hunk 1]
```diff
@@ -50,6 +51,10 @@ impl NetworkVersion {
     pub const V17: Self = Self(17);
     /// Hygge (builtin-actors v10)
     pub const V18: Self = Self(18);
+    /// Lightning (builtin-actors v11)
+    pub const V19: Self = Self(19);
+    /// Lightning (builtin-actors v11)
```
Comment (ZenGround0) [inline shared/src/version/mod.rs:56]: Idk how we're thinking about actors versioning now but wouldn't this be v12?

**Review by ZenGround0 — APPROVED** [review]

File: shared/src/version/mod.rs (hunk 1 above)
Comment (arajasek) [reply shared/src/version/mod.rs:56]: No, because FIP-0061 proposes a 2-stage upgrade, with a rollover period.

Comment (Stebalien) [reply shared/src/version/mod.rs:56]: Call the second one Thunder, maybe?

**Review by Stebalien — APPROVED** [review]

File: shared/src/version/mod.rs (hunk 1 above)
Comment (arajasek) [reply shared/src/version/mod.rs:56]: That's such a good idea, it's worth re-running CI.

**Pull request merged**